recipes/efmtool_link/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = '0.0.7' %}
+
+package:
+  name: efmtool_link
+  version: {{ version }}
+
+source:
+  url: https://github.com/cnapy-org/efmtool_link/archive/v{{ version }}.tar.gz
+  sha256: 8380248c011c20593262a373fc0b7246fde4763c5f28a1b689b8a78442db09f0
+
+build:
+  noarch: python
+  number: 0  # NOTE: increment for new build/set to zero for new version
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - setuptools
+    - python >=3.7,<3.11
+  run:
+    - python >=3.7,<3.11
+    - numpy =1.23
```

Comment:
can you be a bit flexible in this dep, or does it really need to be numpy 1.23?

Reply:
At present we need this because some of the required packages may produce run time errors with other numpy versions (hopefully we can relax this in the future).

Reply:
Actually, this is not entirely correct. I plan to add more recipes for packages that use efmtool_link where this is the case, so I can relax this in the current recipe.